driver-async/src/test/unit/com/mongodb/async/client/AggregateIterableSpecification.groovy

```diff
@@ -139,12 +140,14 @@ class AggregateIterableSpecification extends Specification {
                 .comment('this is a comment'))
 
         when: 'the subsequent read should have the batchSize set'
-        operation = operation.getFindOperation()
+        operation = operation.getReadOperation() as FindOperation
 
         then: 'should use the correct settings'
         operation.getNamespace() == collectionNamespace
+        operation.getCollation() == collation
         operation.getBatchSize() == 99
-        operation.getMaxAwaitTime(MILLISECONDS) == 99
+        operation.getMaxAwaitTime(MILLISECONDS) == 0
```

Comment:
This was a difference in behavior between driver and driver-async.  The driver behavior is correct, so changed the driver-async specification test.

Reply:
I would have thought it shouldn't be ignored for the cursor and sync should have it set?

Reply:
it should be ignored because maxAwaitTime only makes sense for a tailable cursor (and so only makes sense when the first pipeline stage is $changeStream).  Since the FindOperation is not tailable it doesn't apply